models/src_twilio.yml

```diff
@@ -43,8 +43,8 @@ sources:
             description: timestamp of when fivetran synced a record. 
 
           - name: account_id
-            description: >
-              The unique identifier of the account associated with the address.
+            description: > 
+              The unique identifier of the account associated with the address. 
```

Comment:
Not a big issue, but is there a reason there is not consistency with the description format? Some use the `>` to put the description on a new line and others don't. Typically I have only seen us use the `>` if the description is exceptionally long. But these seem pretty short. Can we update the docs to be consistent with the format.

Reply:
Yeah, when I used AI to generate the descriptions, it automatically added them, didn't think it was worth the time to remove it but I did now!